__tests__/app.test.js

```diff
@@ -49,4 +49,37 @@ describe("GET /api",()=>{
             expect(typeof body.endpoints["GET /api"].description).toBe("string")
         })
     })
+})
+
+describe("GET /api/articles/:article_id",()=>{
+    it("should return an object article with the ID passed",()=>{
+        return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({body})=>{
+            expect(typeof body.article).toBe("object")
```

Comment:
because we have access to the test data and we know it's being reseeded before running each test - we can afford to be a little more specific with what we're expecting back here.